nolearn/lasagne/base.py

```diff
@@ -516,9 +527,11 @@ def _create_iter_funcs(self, layers, objective, update, output_type):
 
         return train_iter, eval_iter, predict_iter
 
-    def fit(self, X, y, epochs=None):
+    def fit(self, X, y, labels=None, epochs=None):
```

Comment:
I like this ordering of arguments. The downside is that if we issue a Pull Request for these changes (which I think we should), this will cause existing calls to fit() to break. The compatible way to do it is to add `labels=None` as a final argument, which is unfortunate.